### PR TITLE
MichaelKendall HWD04W07

### PR DESCRIPTION
**Pull request by michaelkendall**
Completeness: 80%
Comfort: 4

### CODE REVIEW

File: timers.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        secs = "0" + secs;
+      }
+      //this updates my output field, this is whats seen on the screen when the stop watch is running
+      document.getElementById("output").innerHTML = hours + ":" + mins + ":" + secs + ":" + tenths; //+"0";
+      //by adding this function inside of my function it makes my stopwatch continue running
+      increment();
+
+
+    },100)//This gives a delay of less than a sec
+  }
```
Comment (andrewsunglaekim) [inline timers.js]: setTimeout basically means you will only run the function once, and that will happen after 100 miliseconds, You want to use `setInterval` instead.

File: timers.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      var secs = Math.floor(time/10 % 60);
+      var hours = Math.floor(time/10/60/60);
+      var tenths = time % 10;
+      //
+      if(mins < 10 {
+        mins = "0" + mins;
+      }
+      if(secs < 10) {
+        secs = "0" + secs;
+      }
```
Comment (andrewsunglaekim) [inline timers.js]: i like the minutes and seconds addition of 0 here for the timer

File: timers.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+{
+  //when user clicks reset, the clock is going to stop running and the time should be rest to 0
+  running = 0;
+  time = 0;
+  //
+  document.getElementById("re").innerHTML = "Start";
+
+  document.getElementById("output").innerHTML = "00:00:00";
+}
+
```
Comment (andrewsunglaekim) [inline timers.js]: the question i want to ask is, what is calling the `start` `pause` and `reset` methods? I think if you add some event listeners you can get an idea of what these functions are doing for you.

**Comment by andrewsunglaekim** [comment]
I think you're on the right track. You need to add some event listeners to see when/where your functions need to occur. If you have any questions let me know, I'll be happy to answer them.

**Comment by michaelkendall** [comment]
I'm just checking my email. Thanks for the feedback, I really appreciate it.

**Pull request closed without merging**